### ISSUE TITLE
v2.2.2 getting invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by szinn** [containous/traefik#7017]
### Do you want to request a *feature* or report a *bug*?

Bug

My log is filled with these with v2.2.2 that was just released.  All was good with 2.2.2.  As well, all internal services report "Internal Server Error"

time="2020-07-09T06:29:24-04:00" level=error msg="Recovered from panic in HTTP handler [192.168.0.160:57358 - /dashboard/]: runtime error: invalid memory address or nil pointer dereference" middlewareType=Recovery middlewareName=traefik-internal-recovery
time="2020-07-09T06:29:24-04:00" level=error msg="Stack: goroutine 229 [running]:\ngithub.com/containous/traefik/v2/pkg/middlewares/recovery.recoverFunc(0x2e33c40, 0xc0006d9680, 0x7fc2007eca30, 0xc000206130, 0xc0005c6900)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:47 +0x1db\npanic(0x242c760, 0x477fb60)\n\t/usr/local/go/src/runtime/panic.go:969 +0x166\ngithub.com/containous/traefik/v2/pk$
time="2020-07-09T06:29:24-04:00" level=error msg="Recovered from panic in HTTP handler [192.168.0.160:57358 - /favicon.ico]: runtime error: invalid memory address or nil pointer dereference" middlewareName=traefik-internal-recovery middlewareType=Recovery
time="2020-07-09T06:29:24-04:00" level=error msg="Stack: goroutine 245 [running]:\ngithub.com/containous/traefik/v2/pkg/middlewares/recovery.recoverFunc(0x2e33c40, 0xc0008230e0, 0x7fc2007eca30, 0xc00011c130, 0xc000414c00)\n\t/go/src/github.com/containous/traefik/pkg/middlewares/recovery/recovery.go:47 +0x1db\npanic(0x242c760, 0x477fb60)\n\t/usr/local/go/src/runtime/panic.go:969 +0x166\ngithub.com/containous/traefik/v2/pk$
time="2020-07-09T06:29:26-04:00" level=error msg="Recovered from panic in HTTP handler [192.168.0.160:57358 - /dashboard/]: runtime error: invalid memory address or nil pointer dereference" middlewareName=traefik-internal-recovery middlewareType=Recovery

### What did you do?

Upgraded from v2.2.1 to v2.2.2

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
(paste your output here)
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
# (paste your configuration here)
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
(paste your output here)
```

**Comment by SeaLife** [comment]
Hey,

we'r using Passtlscert features and we'r encountering the same problems, rollback to v2.2.1 worked just fine.

Regards
SeaLife

**Comment by szinn** [comment]
Yes, that's what I did as well

**Comment by ldez** [comment]
Duplicate of #7014

Fixed by #7016.
